aunt-leahs-app/src/pages/adminLocations.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Head from '../components/header.jsx';
+import constants from '../constants';
+import '../styles.css';
+import { LocationList } from '../components/adminLocationList/locationList';
+import TabComponent from '../components/tab';
+
+const AdminLocationsPage = () => (
+    <div className="homepage">
+        <div style={{display:'flex', flexDirection:'row', alignItems: 'center'}}>
```

Comment:
If ALP-49 is merged before, you can just replace this div with the AdminHeader component

Reply:
Merged in master and used that instead, thanks!